peachjam/tests/test_views.py

```diff
@@ -0,0 +1,114 @@
+from django.test import TestCase
+
+
+class PeachjamViewsTest(TestCase):
+    fixtures = ["documents/sample_documents"]
+
+    def test_login_page(self):
+        response = self.client.get("/accounts/login/")
+        self.assertTemplateUsed(response, "account/login.html")
+
+    def test_homepage(self):
+        response = self.client.get("/")
+        self.assertEqual(response.status_code, 200)
+
+        # documents
+        self.assertEqual(response.context_data["documents_count"], 8)
+        self.assertEqual(len(response.context_data["recent_judgments"]), 2)
```

Comment:
I really advise NOT asserting against the length of the list. When it fails, it's really hard to debug what is missing/in the list, because it just says "2 != 3".

Rather, assert that a particular string is in the page, or turn the list into a string of things (eg. titles) and use `self.assertEqual(["foo", "bar", "baz"], titles)` or similar.